Mathlib/Algebra/Category/ModuleCat/Monoidal/Closed.lean

```diff
@@ -21,20 +21,28 @@ namespace ModuleCat
 
 variable {R : Type u} [CommRing R]
 
+-- Not sure how to define these, presumably there's some abstract machine that does exactly this!
+def asHom₂ {M N P : ModuleCat.{u} R} (f : M →ₗ[R] N →ₗ[R] P) :
+    M ⟶ ((linearCoyoneda R (ModuleCat R)).obj (op N)).obj P := sorry
+
+-- Not sure how to define these, presumably there's some abstract machine that does exactly this!
+def Hom.hom₂ {M N P : ModuleCat.{u} R}
+    (f : ModuleCat.Hom M (((linearCoyoneda R (ModuleCat R)).obj (op N)).obj P)) :
+    M →ₗ[R] N →ₗ[R] P := sorry
```

Comment:
This feels like there is some categorical definition already, but I couldn't find it (without searching very hard).